miso-web/src/main/webapp/scripts/sequencer_partition_container_ajax.js

```diff
@@ -137,17 +137,50 @@ var Container = Container || {
     jQuery('#container-form').parsley();
     jQuery('#container-form').parsley().validate();
     
-    console.log(jQuery('div[id^="studySelectDiv"]'));
+    if(jQuery("#identificationBarcode").length == 0) {
+    	// Serial number is not being modified.
+        Validate.updateWarningOrSubmit('#container-form', Container.validateStudyAdded);
+    } else {
+    	// Ensure provided serial number is unique.
+    	var serialNumber = jQuery("#identificationBarcode").val();
+    	var containerId = null;
+		if(jQuery("#containerId").length > 0) {
+		  containerId = jQuery("#containerId").val();
+		}
+
+	    Fluxion.doAjax(
+        'containerControllerHelperService',
+        'isSerialNumberUnique',
+        {
+          'serialNumber': serialNumber,
+          'containerId': containerId,
+          'url': ajaxurl
+        },
+        {
+          'doOnSuccess': function(json) {
+        	  if(json.isSerialNumberUnique === "true") {
+                  Validate.updateWarningOrSubmit('#container-form', Container.validateStudyAdded);
+        	  } else {
+        		  // Serial number is not unique.
+                  var serialNumberField = jQuery('#identificationBarcode').parsley();
+            	  window.ParsleyUI.addError(serialNumberField, "serialNumberError", 'This serial number is already in use. Please choose another.');
+            	  return false;
+        	  }
+          },
+          'doOnError': function(json) {
+        	// Unable to perform lookup.
+        	alert(json.error);
+        	return false;
```

Comment:
I'm not seeing the line where the parsley error is added on the 'doOnError' condition, am I missing something? (this should also likely go in the 'doOnError' condition in the run_ajax.js file on line 112)